tests/API/XCCDF/unittests/test_xccdf_check_processing_selector_empty.sh

```diff
@@ -12,7 +12,7 @@ $OSCAP xccdf eval --profile xccdf_moc.elpmaxe.www_profile_1 --results $result $s
 
 echo "Stderr file = $stderr"
 echo "Result file = $result"
-[ -f $stderr ]; [ ! -s $stderr ]; rm $stderr
+rm $stderr
```

Comment:
So now the `stderr` should contain the warning message right? What about greping it similarly as it is done in the `tests/API/XCCDF/unittests/test_xccdf_multiple_testresults.sh`?